test/runtests.jl

```diff
@@ -22,7 +22,7 @@ Aqua.test_ambiguities(PowerSystems)
 Aqua.test_stale_deps(PowerSystems)
 Aqua.test_deps_compat(PowerSystems)
 
-include(joinpath(@__DIR__, "../src/utils/data.jl"))
+#include(joinpath(@__DIR__, "../src/utils/data.jl"))
```

Comment:
These lines should have been removed already in PSY 2.0. Do we need to rebase psy3 branch?

Reply:
Yeah the function was repeated two times. I think it was an issue with a rebase.